### PR TITLE
Included several levels of relations in relationships test

### PR DESCRIPTION
**Pull request by chamini2**

### CODE REVIEW

**Comment by jamesdixon** [comment]
Thanks, @chamini2!

File: spec/bookshelf-spec.ts [hunk 1]
```diff
@@ -633,11 +633,13 @@ describe('Bookshelf relations', () => {
     bookshelf.knex.destroy(done);
   });
 
-  it('should add relationships object', () => {
+  fit('should add relationships object', () => {
```
Comment (ShadowManu) [inline spec/bookshelf-spec.ts]: Hey, the word `fit` is wrong in so many levels!

Comment (chamini2) [inline spec/bookshelf-spec.ts]: It stands for _focused it_.

Comment (ShadowManu) [inline spec/bookshelf-spec.ts]: that *was* the point: you were adding a test that commands jasmine to not-to-test all the other ones. Wrong in so many levels. But you ended changing it back and me not noticing. Everyone happy ;)

Comment (chamini2) [inline spec/bookshelf-spec.ts]: It stayed there from some debugging I had to do, and I think you're being a little over the top with that "wrong in so many levels" statement.

**Comment by ShadowManu** [comment]
Well, if you leave it, and we miss it, you're basically disabling the Travis CI, and it could have been hard to get. Easy mistake while using the very useful `fit`, but with effects much more big that one could expect with an extra `f` letter. So yeah, I'll change my statement from `wrong in so many levels` to `with a powerful chance to negatively impact and debug`.

**Pull request merged**